filechooser-portal/FileChooserDialog.vala

```diff
@@ -47,21 +47,21 @@ public class Files.FileChooserDialog : Hdy.Window, Xdp.Request {
         }
     }
 
-    private Hdy.HeaderBar header;
+    private Adw.HeaderBar header;
```

Comment:
Do we need to use Adw.HeaderBar here instead of Gtk.HeaderBar?

Reply:
There's little difference so I think Gtk.HeaderBar will suffice